ci/build-unix.yml

```diff
@@ -40,3 +44,20 @@ steps:
     env:
       # to connect to bintray
       JFROG_CONFIG_CONTENT: $(JFROG_CONFIG_CONTENT)
+  - bash: |
+      set -euxo pipefail
+      git tag v$(release_tag)
+      git push origin v$(release_tag)
+      mv bazel-genfiles/release/sdk-release-tarball.tar.gz ./daml-sdk-$(release_tag)-${{ parameters.name }}.tar.gz
```

Comment:
I think our builders are no longer completely ephemeral right? So maybe a `cp` would be better?

Reply:
Good point. Also, probably a good idea to `rm` afterwards. I'll add a cleanup step.